Client/Frontend/Settings/PasswordDetailViewController.swift

```diff
@@ -129,10 +129,12 @@ extension PasswordDetailViewController: UITableViewDataSource {
             breachDetailView.setup(breach)
             breachDetailView.applyTheme(theme: themeManager.currentTheme)
 
-            breachDetailView.viewBreachDetailsButton.addTarget(self, action: #selector(PasswordDetailViewController.didTapBreachLearnMore), for: .touchUpInside)
+            breachDetailView.onTapLearnMore = { self.didTapBreachLearnMore() }
             let breachLinkGesture = UITapGestureRecognizer(target: self, action: #selector(PasswordDetailViewController
                 .didTapBreachLink(_:)))
-            breachDetailView.visitBreachSource.addGestureRecognizer(breachLinkGesture)
+            breachDetailView.addGestureRecognizer(breachLinkGesture)
```

Comment:
This line it's adding the `breachLinkGesture` to the entire view, and we dont want this. Instead as mentioned eralier, we want to keep `breachDetailView` properties private, so like in the previous example we would also need to add the tap gesture in `goToLabel` property from `BreachAlertsDetailView`. 
`let breachLinkGesture = UITapGestureRecognizer(target: self, action: #selector(self.didTapBreachLink))`
`label.addGestureRecognizer(breachLinkGesture)`
And after we create our selector method and call there the `onTapBreachLink?()` closure.
`@objc
    private func didTapBreachLink() {
        onTapBreachLink?()
    }`